projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -17,30 +17,78 @@
 
 <ng-container *ngIf="record">
   <header class="row">
-    <h1>{{ record.metadata.title }}</h1>
+
+    <!-- TITLE -->
+    <ng-container *ngIf="record.metadata.title">
+      <ng-container *ngFor="let title of record.metadata.title">
+        <ng-container *ngIf="title.type === 'bf:Title'">
+          <h1>{{ title._text }}</h1>
+        </ng-container>
+      </ng-container>
+    </ng-container>
   </header>
   <section>
     <dl class="row mb-0">
+
+      <!-- ALTERNATIVE GRAPHICS -->
+      <ng-container *ngIf="record.metadata.ui_title_altgr">
+        <ng-container *ngFor="let altgr_title of record.metadata.ui_title_altgr; first as isFirst">
```

Comment:
what is ui_title_altgr? Seems weired.

Reply:
Alternative graphics. Comes from backend